core/templates/base-components/base-content.directive.html

```diff
@@ -2,8 +2,9 @@
   <div ng-if="$ctrl.iframed">
     <div tabindex="0" aria-label="Oppia Main Content" id="oppia-main-content" class="protractor-test-main-content"
          ng-cloak>
-      <warning-loader></warning-loader>
-      <div ng-show="!$root.loadingMessage">
+      <loading-message message="$ctrl.loadingMessage"></loading-message>
+      <warning-and-alerts></warning-and-alerts>
```

Comment:
Just one check -- should this be "warnings and alerts" (plural)? Not sure.

Reply:
I will check it

Reply:
@seanlip This can show up to five alerts... should be plural right?

Reply:
My question was different: it was about the "warnings" part. Could you please check?

Reply:
Ah, yes it should be plural, changing it now.